.changelog/15302.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:improvement
+config: update 1.14 config defaults: Enable `peering` and `connect` by default.
```

Comment:
Is there a reason why peering and connect are not breaking changes? I would think it would be breaking as it may cause issues for folks upgrading who previously did not enable those features.

Reply:
@hashi-derek may have more context here. 
@david-yu Are there any specific cases that enabling peering or connect can break things?

Reply:
I think he means to set the tag to `release-note:breaking-change`

Reply:
Sorry, I only saw the conversation and two lines of code in the conversation overview, and I didn't see the whole file here. I think it wouldn't hurt to put it as a breaking change if there are users that explicitly wanted to disable connect / peering for some reason.

I don't think peering will have any adverse side-effects if nobody attempts to use the feature set, but it could be a security concern for operators, I suppose. Connect is complicated, so it could potentially have some undesired effects.

Reply:
@hashi-derek For large service discovery use cases I'm wondering if it does cause issues. I would recommend keeping it at breaking change for connect perhaps unless we have some internal testing that provides evidence that upgrades on large scale SD system on 1.13 looks ok.

Reply:
Yeah, let's just list them both as breaking so that operators are more aware that these are enabled.